report_pre_thesis/src/references.bib

```diff
@@ -103,3 +103,142 @@ @article{knight2000
+	pages = {1729--1749},
+	number = {7},
+	journaltitle = {{IEEE} Transactions on Plasma Science},
+	shortjournal = {{IEEE} Trans. Plasma Sci.},
+	author = {Huang, Yingchao and Harilal, Sivanandan S. and Bais, Abdul and Hussein, Amina E.},
+	urldate = {2023-11-13},
+	date = {2023-07},
+	langid = {english},
+	file = {Huang et al. - 2023 - Progress Toward Machine Learning Methodologies for.pdf:C\:\\Users\\Ivik Hostrup\\Zotero\\storage\\TCQP7565\\Huang et al. - 2023 - Progress Toward Machine Learning Methodologies for.pdf:application/pdf},
+}
+
+@article{huang_progress_2023,
+	title = {Progress Toward Machine Learning Methodologies for Laser-Induced Breakdown Spectroscopy With an Emphasis on Soil Analysis},
+	volume = {51},
+	issn = {0093-3813, 1939-9375},
+	url = {https://ieeexplore.ieee.org/document/10004728/},
+	doi = {10.1109/TPS.2022.3231985},
+	abstract = {Optical emission spectroscopy of laser-produced plasmas, commonly known as laser-induced breakdown spectroscopy ({LIBS}), is an emerging analytical tool for rapid soil analysis. However, speciﬁc challenges with {LIBS} exist, such as matrix effects and quantiﬁcation issues, which require further study in the application of {LIBS}, particularly for the analysis of heterogeneous samples, such as soils. Advancements in the applications of machine learning ({ML}) methods can address some of these issues, advancing the potential for {LIBS} in soil analysis. This article aims to review the progress of {LIBS} application combined with {ML} methods, focusing on methodological approaches used in reducing matrix effect, feature selection, quantiﬁcation analysis, soil classiﬁcation, and self-absorption. The performance of various adopted {ML} approaches is discussed, including their shortcomings and advantages, to provide researchers with a clear picture of the current status of {ML} applications in {LIBS} for improving its analytical capability. The challenges and prospects of {LIBS} development in soil analysis are proposed, offering a path toward future research. This review article emphasizes {ML} tools for {LIBS} soil analysis, which are broadly relevant for other {LIBS} applications.},
+	pages = {1729--1749},
+	number = {7},
+	journaltitle = {{IEEE} Transactions on Plasma Science},
+	shortjournal = {{IEEE} Trans. Plasma Sci.},
+	author = {Huang, Yingchao and Harilal, Sivanandan S. and Bais, Abdul and Hussein, Amina E.},
+	urldate = {2023-11-13},
+	date = {2023-07},
+	langid = {english}
+}
+
```

Comment:
Duplicate + also duplicate with the one you have added in https://github.com/chhoumann/thesis-chemcam/pull/21/files

```suggestion
@article{huang_progress_2023,
	title = {Progress Toward Machine Learning Methodologies for Laser-Induced Breakdown Spectroscopy With an Emphasis on Soil Analysis},
	volume = {51},
	issn = {0093-3813, 1939-9375},
	url = {https://ieeexplore.ieee.org/document/10004728/},
	doi = {10.1109/TPS.2022.3231985},
	abstract = {Optical emission spectroscopy of laser-produced plasmas, commonly known as laser-induced breakdown spectroscopy ({LIBS}), is an emerging analytical tool for rapid soil analysis. However, speciﬁc challenges with {LIBS} exist, such as matrix effects and quantiﬁcation issues, which require further study in the application of {LIBS}, particularly for the analysis of heterogeneous samples, such as soils. Advancements in the applications of machine learning ({ML}) methods can address some of these issues, advancing the potential for {LIBS} in soil analysis. This article aims to review the progress of {LIBS} application combined with {ML} methods, focusing on methodological approaches used in reducing matrix effect, feature selection, quantiﬁcation analysis, soil classiﬁcation, and self-absorption. The performance of various adopted {ML} approaches is discussed, including their shortcomings and advantages, to provide researchers with a clear picture of the current status of {ML} applications in {LIBS} for improving its analytical capability. The challenges and prospects of {LIBS} development in soil analysis are proposed, offering a path toward future research. This review article emphasizes {ML} tools for {LIBS} soil analysis, which are broadly relevant for other {LIBS} applications.},
	pages = {1729--1749},
	number = {7},
	journaltitle = {{IEEE} Transactions on Plasma Science},
	shortjournal = {{IEEE} Trans. Plasma Sci.},
	author = {Huang, Yingchao and Harilal, Sivanandan S. and Bais, Abdul and Hussein, Amina E.},
	urldate = {2023-11-13},
	date = {2023-07},
	langid = {english},
}

```